collections/_posts/2022-11-23-Multi Channel Stochastic Variational Inference.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: review
+title: "Multi-Channel Stochastic Variationnal Inference"
+tags: "Fusion, Multimodality, Machine Learning"
+author: "Romain Deleat-besson"
+cite:
+    authors: "Luigi Antelmi, Nicholas Ayache, Philippe Robert and Marco Lorenzi"
+    title:   "Multi-Channel Stochastic Variational Inference for the Joint Analysis of Heterogeneous Biomedical Data in Alzheimer's Disease"
+    venue:   "arXiv 2018"
```

Comment:
It seems the article was published in a MICCAI worshop, so you should put that instead of arXiv:
```suggestion
    venue:   "Machine Learning in Clinical Neuroimaging (MLCN), MICCAI Workshop 2018"
```